src/lib/viewers/media/DashViewer.js

```diff
@@ -166,9 +166,13 @@ class DashViewer extends VideoBaseViewer {
             }
         });
         this.player.getNetworkingEngine().registerRequestFilter(this.requestFilter);
-
         this.startLoadTimer();
-        this.player.load(this.mediaUrl);
+        return this.player.load(this.mediaUrl).catch((error) => {
+            // The error is of a different format than the shakaErrorHandler expects
```

Comment:
I noticed this as well, itt's so weird that shaka formats this error differently.